src/main/java/net/sf/jabref/gui/actions/MassSetFieldAction.java

```diff
@@ -321,6 +329,9 @@ private static UndoableEdit massRenameField(Collection<BibEntry> entries, String
     }
 
     private static String[] getFieldNames(String s) {
+        if (s == null) {
```

Comment:
I will remove this once I rebase because of the translation conflicts as it is not needed...